docs/getstarted.rst

```diff
@@ -41,10 +41,31 @@ It only needs two easy steps:
 .. tip:: Set ``export CAPSULE_LOG=none`` to make the dependency installation logs less verbose.
 
 .. note::
-    To avoid downloading the dependencies, you could also use the ``nextflow-VERSION-all`` variant available from Github for every Nextflow release version.
+    To avoid downloading the dependencies, you could also use the ``nextflow-VERSION-all`` variant available from Github for every `Nextflow` release version.
    #. Go to the `Github releases page <https://github.com/nextflow-io/nextflow/releases>`__ and unfold the `Assets` section for a release.
    #. Copy the URL of the ``nextflow-VERSION-all`` asset and issue the download command on your terminal. ``wget -qO- ASSET-URL``. It will create the completely self-contained ``nextflow-VERSION-all`` executable file in the current directory.
 
+.. _getstart-edge:
+
+Edge releases
+============
+
+Stable releases will be becoming more infrequent as `Nextflow` shifts its development model to becoming more dynamic via the usage of plugins. 
+This will allow functionality to be added as an extension to the core codebase with a release cycle that could potentially be independent to that of `Nextflow` itself. 
+As a result of the reduction in stable releases, you may be required to use `Nextflow` ``edge`` releases in order to be able to exploit cutting "edge" features.
+
+There are a number of ways you can install `Nextflow` ``edge`` releases, the main difference with stable releases being that you have to ``export`` the version you would like to install before issuing the appropriate installation/execution commands as highlighted below:
+
+#.  If you have `Nextflow` installed already, you can issue the version you would like to use on the same line as the pipeline command and it will be fetched if required before the pipeline execution:
+    ``NXF_VER="20.11.0-edge" nextflow run tutorial.nf``.
```

Comment:
Perhaps we should use the latest `edge` release as an example?

Reply:
Absolutely! Good idea :) Fixed https://github.com/nextflow-io/nextflow/pull/2077/commits/e5d762aef5e02618c51a663b568c426689fd4ab4